relayer/packet-tx.go

```diff
@@ -135,16 +136,16 @@ func (src *Chain) SendTransferBothSides(dst *Chain, amount sdk.Coin, dstAddr sdk
 		Src: []sdk.Msg{},
 	}
 
-	txs.Send(src, dst)
+	txs.Send(c, dst)
 	return nil
 }
 
 // SendTransferMsg initiates an ibs20 transfer from src to dst with the specified args
-func (src *Chain) SendTransferMsg(dst *Chain, amount sdk.Coin, dstAddr sdk.AccAddress, source bool) error {
+func (c *Chain) SendTransferMsg(dst *Chain, amount sdk.Coin, dstAddr fmt.Stringer, source bool) error {
```

Comment:
why the change from sdk.AccAddress to fmt.Stringer?

Reply:
this was due to the intefacer linter. Im not sure if this is a false positive or if we want to remove that linter

Reply:
I think it should be fine for now